installation/docker-commands/update.sh

```diff
@@ -105,23 +105,19 @@ execute_docker () {
    docker run -itd --log-opt max-size=10m --log-opt max-file=5 \
    --network host \
    --name ${INSTANCES[$j]} \
-   --mount "type=bind,source=${FOLDERS[$j]}/hummingbot_conf,destination=/conf/" \
-   --mount "type=bind,source=${FOLDERS[$j]}/hummingbot_logs,destination=/logs/" \
-   --mount "type=bind,source=${FOLDERS[$j]}/hummingbot_data,destination=/data/" \
-   --mount "type=bind,source=${FOLDERS[$j]}/hummingbot_scripts,destination=/scripts/" \
-   --mount "type=bind,source=${FOLDERS[$j]}/hummingbot_pmm_scripts,destination=/pmm_scripts/" \
-   --mount "type=bind,source=${FOLDERS[$j]}/hummingbot_certs,destination=/home/hummingbot/.hummingbot-gateway/certs/" \
-   --mount "type=bind,source=${FOLDERS[$j]}/gateway_conf,destination=/gateway-conf/" \
-   --mount "type=bind,source=${FOLDERS[$j]}/gateway_logs,destination=/gateway_logs/" \
+   --mount "type=bind,source=${FOLDERS[$j]}/conf,destination=/conf/" \
+   --mount "type=bind,source=${FOLDERS[$j]}/logs,destination=/logs/" \
+   --mount "type=bind,source=${FOLDERS[$j]}/data,destination=/data/" \
+   --mount "type=bind,source=${FOLDERS[$j]}/scripts,destination=/scripts/" \
+   --mount "type=bind,source=${FOLDERS[$j]}/pmm-scripts,destination=/pmm-scripts/" \
```

Comment:
review the start.sh script, we changed the mounts for volumes and also we removed the environment variables. Copy the structure of start.sh and we are ready to go!